Fix for #63318

<+>UTF-8
===================================================================
--- vendor/symfony/serializer/Normalizer/AbstractObjectNormalizer.php
+++ vendor/symfony/serializer/Normalizer/AbstractObjectNormalizer.php
@@ -410,7 +411,7 @@
             } catch (PropertyAccessInvalidArgumentException $e) {
                 $exception = NotNormalizableValueException::createForUnexpectedDataType(
                     \sprintf('Failed to denormalize attribute "%s" value for class "%s": '.$e->getMessage(), $attribute, $resolvedClass),
-                    $data,
+                    $data[$attribute] ?? $value,
                     $e instanceof InvalidTypeException ? [$e->expectedType] : ['unknown'],
                     $attributeContext['deserialization_path'] ?? null,
                     false,

